src/libs/actions/PaymentMethods.js

```diff
@@ -27,4 +31,48 @@ function getPaymentMethods() {
         });
 }
 
-export default getPaymentMethods;
+/**
+ * Calls the API to add a new card.
+ *
+ * @param {Object} params
+ */
+function addBillingCard(params) {
+    API.AddBillingCard({
+        cardNumber: params.cardNumber,
+        // eslint-disable-next-line no-use-before-define
+        cardYear: normalizeCardYear(params.expirationDate),
+        cardMonth: params.expirationDate.substr(0, 2),
+        cardCVV: params.securityCode,
+        addressName: params.nameOnCard,
+        addressZip: params.zipCode,
+        currency: CONST.CURRENCY.USD,
+    }).then(((response) => {
+        if (response.jsonCode === 200) {
+            Onyx.set(ONYXKEYS.CARD_LIST, response);
+            Growl.show(translateLocal('addDebitCardPage.growlMessageOnSave'), CONST.GROWL.SUCCESS, 3000);
+            Navigation.navigate(ROUTES.SETTINGS_PAYMENTS);
+        } else {
+            Growl.error(translateLocal('addDebitCardPage.error.genericFailureMessage', 3000));
+        }
+    }));
+}
+
+/**
+ * Returns the year of the expiration date in YYYY format.
+ *
+ * @param {String} expirationDate
+ *
+ * @returns {String}
+ */
+function normalizeCardYear(expirationDate) {
+    let cardYear = expirationDate.substr(3);
+    if (cardYear.length === 2) {
+        cardYear = `20${cardYear}`;
+    }
+    return cardYear;
+}
```

Comment:
I'm not a big fan of this, I don't think we should be modifying user input before sending it to the API.

If we need a 4 digit year then let's just update the regex, placeholder, and input to all expect a 4 digit year and have the user input it themselves. 

Having all this extra code is cumbersome and will be harder to debug than just asking the user to input 2 extra digits.

Reply:
The first implementation followed the specs (2 digits), but I was asked to change it to 2 or 4 digits (and to convert from 2 to 4). I'll wait for @stitesExpensify to reply before changing this yet again

Reply:
I personally like doing both. Just because some cards have the date written on them as `mm/yyyy` and some are just `mm/yy` (I think most are in the latter format). Our API requires a 4 digit year but I don't see a need to pass that requirement onto the user. 

I'm not really worried about the debugging either since this function is so simple. Thoughts @bondydaa?

Reply:
I mean this is not easy code to follow IMO and add so much complexity for such little gain - it's not hard for someone to enter 2021 vs 21 when reading it off their card, specially if we're just requiring you do enter 4 digits. Just be we _can_ code something doesn't mean we _should_.

The same result can be achieved by ensuring MM/YYYY with a `pattern="\d{2}\/\d{4}":` on the input which reduces the complexity of the expiration regex in the const file (or allows us to remove it all together). If a user screws up entering their expiration date then we'll return that as a nice error for them to correct and it's on them.

If we have that client side validation then we remove this normalize method all together then which means we're not editing a user's input before it's submitted and have reduced the amount of code we need to maintain in the future.

If you give users wiggle room to fuck something up they will and if we're editing what they input then our logs will show something different than what they gave us and it won't be clear from logs that we edited their entry unless you look at the code. 

Ultimately I'm not going to be the one to maintain any of this code so don't consider this a blocker (it will just be another reason I won't want to touch this code base 😅).

Reply:
Alright, sorry for the back and forth @rdjuric but let's just require `MM/YYYY` to simplify the code. After that I'm thinking this is good to go!

Reply:
No worries! It's part of the process 😄